### ISSUE TITLE
Pull Request body is not updated when updatecli updates the PR.

### ISSUE THREAD

**Issue by quotidian-ennui** [updatecli/updatecli#1784]
### Is there an existing issue for this?

- [X] I have searched the existing issues

## Sequence of events

- A PR was originally created by updatecli for updating localstack to 3.0.0
- We never merged this PR (because it's a major upgrade) from 2.3.2
- A new release of localstack is now available (3.0.1)
- updatecli re-ran ( weekly schedule).

## What is changed in the updated PR.

- ✅ -> the PR title changes to reflect the new update
- ✅ -> The commit history in the PR shows all the changes
- ✅ -> The workflow run link points to the _latest execution_ (i.e. update 3.0.0 -> 3.0.1)

## What is wrong in the updated PR

- The PR body has not changed to reflect the new version in the targets.

### Expected Behavior

Since you're editing everything else in the PR body; the PR body should reflect the latest change made.

### Environment

```markdown
- OS: ubuntu-latest (github-action)
- updatecli: v0.67.0 (it ran before updatecli updated itself).
```

### Anything else?

The updated PR body basically contains the changes from the original PR creation

```
localstack
Update setup-localstack action

key "$.inputs.version.default" updated from "2.3.2" to "3.0.0", in file ".github/actions/setup-localstack/action.yml"
Update localstack docker-compose default

key "$.services.localstack_http.image" updated from "localstack/localstack:2.3.2" to "localstack/localstack:3.0.0", in file "localstack/docker-compose.yml"

```

The workflow run log attached to the updated PR shows us that we are updating from 3.0.0 to 3.0.1

```
##############
# LOCALSTACK #
##############


SOURCES
=======

localstack-latest
-----------------
Searching for version matching pattern "*"
✔ value "3.0.1", found in file "https://pypi.org/pypi/localstack/json", for key ".releases.-"


CONDITIONS:
===========

localstack-docker
-----------------
✔ docker image docker.io/localstack/localstack:3.0.1 found


TARGETS
========

localstack-compose-update
-------------------------
⚠ - key "$.services.localstack_http.image" updated from "localstack/localstack:3.0.0" to "localstack/localstack:3.0.1", in file "localstack/docker-compose.yml"

localstack-action-update
------------------------
⚠ - key "$.inputs.version.default" updated from "3.0.0" to "3.0.1", in file ".github/actions/setup-localstack/action.yml"
```

**Comment by quotidian-ennui** [comment]
This is largely a cosmetic bug in my opinion, the changes are hidden behind a expansion and the title/commit history shows the right thing.

**Comment by olblak** [comment]
Thanks for reporting, I thought that everything was updated.
I'll keep an eye on this issue when I merge long running pullrequest

**Comment by quotidian-ennui** [comment]
c.f. https://github.com/quotidian-ennui/ubuntu-dpm/pull/24

- title is updated to 0.30.4
- new 0.30.4 release notes included in the PR
- the "change detected" is not updated; it still sits at 0.29.x -> 0.30.0

**Comment by olblak** [comment]
Theoretically this issues should be fixed by https://github.com/updatecli/updatecli/pull/2021

And more specifically in https://github.com/updatecli/updatecli/blob/951bc936b8177ed559422a7ebd0d6a7636e97b73/pkg/plugins/scms/github/pullrequest.go#L571 where I revert the order of the merge action so the new report should take precedence over the old one

**Comment by quotidian-ennui** [comment]
https://github.com/quotidian-ennui/interlok-build-parent/pull/114 still illuminates the issue (updatecli-0.7.4 execution); so lets see what happens when you release 0.75 & gradle-8.8 is released

**Comment by quotidian-ennui** [comment]
Yep it's fixed by your recent changes, it changes the description from "original version" -> "latest version" so it's fixed in 0.75
Closing this ticket.
